code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm

```diff
@@ -194,12 +212,26 @@ Difficulty: Hard
 	else //just release a burst of power
 		INVOKE_ASYNC(src, PROC_REF(burst), get_turf(src))
 
+/mob/living/simple_animal/hostile/megafauna/hierophant/proc/easy_anti_cheese(mob/living/simple_animal/S)
+	if(enraged || get_dist(S, src) > 20)
+		return
+	for(var/mob/living/L in urange(20, src))
+		if(L.client)
+			enrage()
+			arena_trap(L, TRUE)
+			FindTarget(list(L), 1)
+			for(var/mob/living/simple_animal/hostile/megafauna/colossus/C in GLOB.mob_list)
+				UnregisterSignal(C, COMSIG_MOB_DEATH)
```

Comment:
Isnt this meant to remove the register for damage? I dont think you every applied one for death
```suggestion
				UnregisterSignal(C, COMSIG_MOB_APPLY_DAMAGE)
```

Reply:
whoops